README.md

```diff
@@ -8,10 +8,15 @@ Note: Binary is just launching server (similar to `signalhub listen -p $PORT`)
 
 See https://github.com/mafintosh/signalhub
 
-## Deploy with now
+## Deploy with Heroku
 
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
+	
+Or via the CLI:
```

Comment:
Should one be inside the signalhubws directory to run these commands? We should add this information, for example:

> Or via the CLI once cloned, go to the signalhubws directory and run: